### DISCUSSION TITLE
Sonoff ZBmini

### DISCUSSION THREAD

**Discussion by tzeryin1981** [Koenkk/zigbee2mqtt#14096] [Devices]
I had 25 numbers of sonoff zbmini installed in my house, controlling the lighting. The devices work flawless 1st few months, however it started not responsive after.
It will response the 1st toggle on/off, but not response immediately to the following 2nd or 3rd toggle. And it came out an error command msg, and only response after few seconds. 
It only happened to the sonoff, but not other brand, anybody know the reason? I had adjust different zigbee channel with the wifi. 
If one of the sonoff start behave slow, it also affect other device. Remove the problem device, other device response very fast.

**Comment by johnypony3** [comment]
same type of issue. this happens when the unit is powered off. trying to figure out if power on/off state can be of use.
